harness/determined/cli/shell.py

```diff
@@ -78,6 +79,11 @@ def open_shell(args: Namespace) -> None:
 
 @authentication.required
 def show_ssh_command(args: Namespace) -> None:
+    if "WSL" in os.uname().release:
+        print(
+            f"WARNING: WSL remote-ssh integration is not supported in VSCode. "
+            "For Windows VSCode integration, rerun this command in a Windows shell."
+        )
```

Comment:
use `colored()` to make this red?  The ssh command already renders as yellow, so having this be default color would make it easy to miss.

Also, please print to stderr, since this is a warning message.  It would be weird to put it on stdout.